### DISCUSSION TITLE
E-mail template language

### DISCUSSION THREAD

**Discussion by sonegomaximiliano** [dani-garcia/vaultwarden#3141] [Ideas]
Add the option to use different languages ​​in the email templates

**Comment by stefan0xC** [comment]
That option already exists. If you copy the handlebar [files](https://github.com/dani-garcia/vaultwarden/tree/main/src/static/templates/email) in `src/static/templates/email` to a `data/templates/email` folder, you can translate them to a different language. After restarting vaultwarden your server should send mails in the new language (assuming that you have not changed the DATA_FOLDER or set a TEMPLATES_FOLDER, otherwise the folder would have to be changed accordingly).
